app/assets/images/chevron-black.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Unfortunately not, I checked and the designers need the icon to be black for these types of accordions.